embrace-android-sdk/lint-baseline.xml

```diff
@@ -67,6 +67,17 @@
             column="20"/>
     </issue>
 
+    <issue
```

Comment:
Is this something we should get rid of?

Reply:
Yep - updated to make the property non-private